Sources/Linq2Acad/AcadDatabase.cs

```diff
@@ -10,6 +10,57 @@
 
 namespace Linq2Acad
 {
+  /// <summary>
+  /// DWG version enum.
+  /// </summary>
+  public enum SaveAsDwgVersion
+  {
+    None,
+    DWG2004,
+    DWG2007,
+    DWG2010,
+    DWG2013,
+#if AutoCAD_2018 || AutoCAD_2019 || AutoCAD_2020 || AutoCAD_2021
+    DWG2018,
+#endif
+    DontChange, // Better than Current? Or Keep? EDIT: We can use Database.LastSavedAsVersion for that
+    NewestAvailable // We could remove this one, since the newest available is AutoCad20xx wit hthe higehst number
+  }
+
+  public class CreateOptions
+  {
+    public bool SaveFileOnCommit { get; set; }
+
+    public string SaveAsFileName { get; set; }
+
+    public SaveAsDwgVersion DwgVersion { get; set; }
```

Comment:
Just a thought: could in the context of creating an in-memory database `DwgVersion` falsely be understood by a user as 'Creating the database in this DwgVersion' instead of 'Saving the database as a DWG in this DwgVersion'?

Reply:
Well, it could be the case.
(From a quick search I cannot find the default behavior, but I assume a newly created database uses the same DWG version as the host application, is this right?)
Should we also name the property `SaveAsDwgVersion`?